api4/team.go

```diff
@@ -21,7 +21,7 @@ func InitTeam() {
 	BaseRoutes.Team.Handle("", ApiSessionRequired(getTeam)).Methods("GET")
 
 	BaseRoutes.TeamMember.Handle("", ApiSessionRequired(getTeamMember)).Methods("GET")
-
+	BaseRoutes.TeamStats.Handle("", ApiSessionRequired(getTeamStats)).Methods("GET")
```

Comment:
Per my above comment, this would become `BaseRoutes.Team.Handle("/stats", ApiSessionRequired(getTeamStats)).Methods("GET")`